src/qumin/entropy/distribution.py

```diff
@@ -119,7 +188,7 @@ def split_if_multiple(preds):
 
         data = pd.read_csv(filename)
         data.loc[:, "predictor"] = data.loc[:, "predictor"].apply(split_if_multiple)
-        self.data = pd.concat(self.data, data)
+        self.concat_data(data)
```

Comment:
This method name is really vague, not clear from this what it does or why it's a method.